cmd/runtimetest/main.go

```diff
@@ -271,6 +271,62 @@ func validateDefaultFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateLinuxDevices(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux devices")
+
+	for _, device := range spec.Linux.Devices {
+		fi, err := os.Stat(device.Path)
+		if err != nil {
+			return err
+		}
```

Comment:
On 09/14/2016 01:52 AM, W. Trevor King wrote:
> you could remove the device before calling |start|
Sorry, I'm not sure about this.
Could you tell me by how to remove the device?